src/mixins/filing-mixin.ts

```diff
@@ -128,7 +130,8 @@ export default class FilingMixin extends Vue {
       business: {
         legalType: this.getEntityType,
         identifier: this.getEntityIncNo,
-        legalName: this.entityName
+        legalName: this.entityName,
+        foundingDate: this.dateToApi(this.getEntityFoundingDate)
```

Comment:
Why does the filing need to tell the API about the founding state, since the API already knows this?

Reply:
Yes, I realize all other filings save this in the business block. 🤷‍♂️

Reply:
> This PR is fine.
> 
> I do note that some of the other filings in this project are without interfaces (or incomplete interfaces). Maybe we can fix that up next time one of us is in that area.
> 
> Also, now that we have a getter for Entity Founding Date, the instances of "state access" can be replaced with the getter. Next time!

ok, sounds good.  Thanks for reviewing.  I'll let the PR stick around a little longer next time.  Trying to rush things a bit maybe.